BUILD.gn

```diff
@@ -219,7 +216,26 @@ static_library("libdeno") {
     "libdeno/internal.h",
   ]
   public_deps = [
-    "third_party/v8:v8_monolith",
+    "third_party/v8:v8",
+    "third_party/v8:v8_libbase",
+    "third_party/v8:v8_libplatform",
+    "third_party/v8:v8_libsampler",
+    "//build/win:default_exe_manifest",
+  ]
+  configs += [ ":deno_config" ]
+}
+
+static_library("deno_deps") {
+  complete_static_lib = true
+  public_deps = [
+    ":libdeno",
+    ":msg_rs",
+    ":snapshot",
+    "third_party/v8:v8",
```

Comment:
Redundant? libdeno alreadu includes v8 in it's public deps.

Reply:
Fixed. (This is part of the work that needs to be done to make `cargo build` work. But that will be put off until a later PR.)